functional-tests/pom.xml

```diff
@@ -39,32 +39,44 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>com.jayway.jsonpath</groupId>
-                <artifactId>json-path</artifactId>
-                <version>2.2.0</version>
+                <groupId>com.google.code.gson</groupId>
```

Comment:
Why do we need gson here in dependencyManagement section? Isn't it already specified in parent pom.